api/nakadi-event-bus-api.yaml

```diff
@@ -799,6 +799,103 @@ paths:
             $ref: '#/definitions/Problem'
 
 
+  /subscriptions/{subscription_id}/offsets:
+    get:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: Exposes the currently committed offsets of a subscription.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
+        '204':
+          description: |
+            No Content. If the reading from the subscription was not started yet - there is nothing to expose.
```

Comment:
I would argue that we should expose an empty list. I say that because then it is more homogeneous. If you program a client oriented to lists, handling both lists with or without something should fit in the same case, making it simpler.